backend/dataall/modules/shares_base/db/share_object_state_machines.py

```diff
@@ -203,6 +203,7 @@ def __init__(self, state):
                     ShareItemStatus.Revoke_Failed.value: [
                         ShareItemStatus.Revoke_In_Progress.value,
                         ShareItemStatus.Revoke_Approved.value,
+                        ShareItemStatus.Revoke_Succeeded.value,
```

Comment:
This is allowing an item that is in `Revoke_Succeeded` to transition to `Revoke_Failed` on `ShareItemActions.Failure` correct?

Should that be the case? I would think `Revoke_Failed` be a state that is unreachable from `Revoke_Succeeded`?

Reply:
See comment below!